packages/components/src/dropdown-menu/style.scss

```diff
@@ -106,4 +106,12 @@
 			padding-left: 0.5rem;
 		}
 	}
+
+	.components-dropdown-menu__menu-item,
+	.components-menu-item__button {
+		&:disabled,
+		&[aria-disabled="true"] {
+			opacity: 0.3;
```

Comment:
Was this fixed in a more global way @aduth? Should we revert that change in that case as it becomes unnecessary styles?

Reply:
Yes, this treats a symptom of the root issue which is that, as of #16103 (https://github.com/WordPress/gutenberg/pull/16103#discussion_r305804002), we have no disabled stying for buttons.

I think the changes there (at least this one line) should be reverted, or find another way which treats _exceptions_ to default disabled styling.

Reply:
See #16769

Reply:
See #16769